source/wp-content/themes/wporg-showcase-2022/src/site-breadcrumbs/index.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * Block Name: Site Breadcrumbs
+ * Description: Display breadcrumbs of the site.
+ *
+ * @package wporg
+ */
+
+namespace WordPressdotorg\Theme\Showcase_2022\Site_Breadcrumbs;
+
+add_action( 'init', __NAMESPACE__ . '\init' );
+
+/**
+ * Registers the block using the metadata loaded from the `block.json` file.
+ * Behind the scenes, it registers also all assets so they can be enqueued
+ * through the block editor in the corresponding context.
+ *
+ * @see https://developer.wordpress.org/reference/functions/register_block_type/
+ */
+function init() {
+	register_block_type(
+		dirname( dirname( __DIR__ ) ) . '/build/site-breadcrumbs',
+		array(
+			'render_callback' => __NAMESPACE__ . '\render',
+		)
+	);
+}
+
+/**
+ * Render the block content.
+ *
+ * @param array    $attributes Block attributes.
+ * @param string   $content    Block default content.
+ * @param WP_Block $block      Block instance.
+ *
+ * @return string Returns the block markup.
+ */
+function render( $attributes, $content, $block ) {
+	if ( ! isset( $block->context['postId'] ) ) {
+		return '';
+	}
+
+	global $pagename;
+	$post_ID = $block->context['postId'];
+	$post = get_post( $post_ID );
+
+	$title = $post->post_title;
+	if ( $pagename ) {
+		$title = ucfirst( strtolower( $pagename ) );
```

Comment:
I think we'll want to use the post title as opposed to the page name. 
For example, on `/Submit-a-wordpress-site` the breadcrumb shows the `-` when it shouldn't.